package.json

```
@@ -1,14 +1,22 @@
 {
   "name": "postcss-rename",
-  "version": "0.7.1",
-  "description": "A PostCSS plugin to replace class names based on a customizable renaming scheme.",
+  "version": "0.8.0-dev",
```

I think this PR is in a good state to release as 0.8.0. So you can update this, and we can do the release right after :)

```suggestion
  "version": "0.8.0",
```